reporter/src/main/kotlin/DefaultResolutionProvider.kt

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright (C) 2017-2018 HERE Europe B.V.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ * License-Filename: LICENSE
+ */
+
+package com.here.ort.reporter
+
+import com.here.ort.model.Error
+import com.here.ort.model.config.Resolutions
+
+class DefaultResolutionProvider : ResolutionProvider {
```

Comment:
Should this maybe more explicitly be called `ResolutionProviderCollection` or so, to describe its purpose in the name somewhat?

Reply:
I find this other name also a bit confusing and couldn't think of a better one. Let's keep it for now.